user-guide/Advanced_Modules/Dashboards_app/Available_visualizations/Charts/Line_and_area_chart.md

```diff
@@ -104,6 +107,23 @@ To configure the component:
 
    - *Additional lines* > *Show boundary lines*: Allows you to display one or more boundary lines. You can configure where a line should be displayed by adding a Y-axis value in the *Boundary value* box. The color of a boundary line can be customized in the *Boundary line color* box. The *Add boundary line* option below this allows you to add additional boundary lines. To remove a boundary line, click the x to the right of the line name.
 
+## Configuration with query data feed
+
+Query results are supported as a data feed for this component from DataMiner 10.2.9/10.3.0 onwards. To configure the component to use a GQI query as its data feed:
+
+1. Create a query data feed. See [Configuring query data feeds](xref:Configuring_GQI_feeds).
+
+1. In the *Component* > *Settings* tab, configure the following fields in the *Dimensions* section:
+
+   - *Query*: The query data feed you want to use.
+   - *X axis*: The column that should be used for the X-axis data.
+   - *Y axis*: The column that should be used for the Y-axis data.
+
+   > [!NOTE]
+   > If you want to show a trend chart, make sure the query results are sorted on the X-axis column.
```

Comment:
Maybe we should make this note a bit more general and give some more details.
I.e. something like:

`Data points are connected by a line in the order they appear in the query result. If you want to create a trend line, make sure the query results are sorted on the desired axis column.`